flair/data.py

```diff
@@ -1434,25 +1434,25 @@ def make_label_dictionary(self, label_type: str, min_count: int = -1, add_unk: b
 
             # go through all labels of label_type and count values
             labels = sentence.get_labels(label_type)
-            for label in labels:
-                if label.value not in all_sentence_labels:
-                    label_value_counter[label.value] += 1
-
-                # check if there are any span labels
-                if type(label.data_point) == Span and len(label.data_point) > 1:
-                    label_dictionary.span_labels = True
+            label_value_counter.update(label.value for label in labels if label.value not in all_sentence_labels)
+            if not label_dictionary.span_labels:
+                for label in labels:
+                    # check if there are any span labels
+                    if type(label.data_point) == Span and len(label.data_point) > 1:
+                        label_dictionary.span_labels = True
+                        break
```

Comment:
is this to speed up the code or another reason? if so, could you add a quick description of non-unit test changes to the issue description?

Reply:
This is for speeding up the code. 
here the change is basically to use `Counter.update()` instead of treating the counter as a dictionary and count manually.